libraries/ESP8266WiFi/src/ESP8266WiFiMulti.h

```diff
@@ -47,6 +47,8 @@ struct WifiAPEntry {
 
 typedef std::vector<WifiAPEntry> WifiAPlist;
 
+struct BestNetwork;

```

Comment:
If selectBest is removed from the class and made a static free standing function, there is no need to declare this here, because the definition in the .cpp is enough. Please remove.